src/pages/dar_application/ResearcherInfo_new.js

```diff
@@ -0,0 +1,641 @@
+    isCloudUseInvalid,
+    isCloudProviderInvalid,
+    isAnvilUseInvalid,
+    ariaLevel = 2
+  } = props;
+
+  const navButtonContainerStyle = {
+    marginTop: '5rem'
+  };
+
+  const formatSOString = (name, email) => {
+    if(isEmpty(name)) { return '';}
+    const nameString = `${name}`;
+    const emailString = !isNil(email) ? ` (${email})` : '';
+    return nameString + emailString;
+  };
+
+  //initial state variable assignment
+  const [checkCollaborator, setCheckCollaborator] = useState(props.checkCollaborator);//
+  const [checkNihDataOnly, setCheckNihDataOnly] = useState(props.checkNihDataOnly);//
+  const [signingOfficial, setSigningOfficial] = useState();//
+  const [itDirector, setITDirector] = useState(props.itDirector || '');//
+  const [principalInvestigator, setPrincipalInvestigator] = useState(props.principalInvestigator || '');//
+  const [anvilUse, setAnvilUse] = useState(props.anvilUse || '');
+  const [cloudUse, setCloudUse] = useState(props.cloudUse || '');
+  const [localUse, setLocalUse] = useState(props.localUse || '');
+  const [researcherUser, setResearcherUser] = useState(props.researcherUser);//
+  const [libraryCardReqSatisfied, setLibraryCardReqSatisfied] = useState(false);
+
+  useEffect(() => {
+    setLibraryCardReqSatisfied(!isEmpty(get('libraryCards')(researcherUser)) || checkNihDataOnly);
+  }, [checkNihDataOnly, researcherUser]);
+
+  useEffect(() => {
+    setSigningOfficial(props.signingOfficial);
+    setCheckCollaborator(props.checkCollaborator);
+    setCheckNihDataOnly(props.checkNihDataOnly);
+    setITDirector(props.itDirector);
+    setPrincipalInvestigator(props.principalInvestigator);
+    setAnvilUse(props.anvilUse);
+    setCloudUse(props.cloudUse);
+    setLocalUse(props.localUse);
+    setResearcherUser(props.researcherUser);
+  }, [props.signingOfficial, props.checkCollaborator, props.itDirector, props.principalInvestigator, props.anvilUse, props.cloudUse, props.localUse, props.researcherUser, props.checkNihDataOnly]);
+
+  const cloudRadioGroup = div({
+    className: 'radio-inline',
+    style: {
+      marginBottom: '2rem',
+      backgroundColor: showValidationMessages && isAnvilUseInvalid ? 'rgba(243, 73, 73, 0.19)' : 'inherit'
+    }
+  }, [
+    [{label: 'Yes', value: true}, {label: 'No', value: false}].map((option) =>
+      label({
+        className: 'radio-wrapper',
+        key: `anvil-use-option-${option.value}`,
+        id: `lbl-anvil-use-option-${option.value}`,
+        htmlFor: `rad-anvil-use-option-${option.value}`
+      }, [
+        input({
+          type: 'radio',
+          id: `rad-anvil-use-option-${option.value}`,
+          name: 'anvil-use-approval-status',
+          checked: option.value === anvilUse,
+          value: option.value,
+          onChange: () => (formFieldChange({name: 'anvilUse', value: option.value}))
+        }),
+        span({ className: 'radio-check'}),
+        span({ className: 'radio-label'}, [option.label])
+      ])
+    )
+  ]);
```

Comment:
could this use the formfield component that I added?

Reply:
it could! I didn't want to rewrite it, and figured it would be in soon! XD